k8s/overlays/.template/kustomization.yaml

```diff
@@ -3,6 +3,8 @@ kind: Kustomization
 
 resources:
   - ../../base
+
+patchesStrategicMerge:
   - cert-issuer.yml
```

Comment:
If you want to check, run `make k8s-build-config | grep letsencrypt-jitsi-issuer -C10`. You should see exactly one `Issuer` named `letsencrypt-jitsi-issuer`.